man/oci-image-tool-create.1.md

```diff
@@ -18,7 +18,9 @@ runtime-spec-compatible `dest/config.json`.
   Print usage statement
 
 **--ref**=""
-  The ref pointing to the manifest of the OCI image. This must be present in the "refs" subdirectory of the image. (default "v1.0")
+  Specify the search criteria, format is A==B.
+  e.g. --ref org.opencontainers.image.ref.name==v1.0
+  Only support `org.opencontainers.image.ref.name`, `platform.os` and `digest` three cases.(default org.opencontainers.image.ref.name==v1.0)
```

Comment:
use `==` seems odd to me, can we just use `=`?  And another question, does it support the old way `--ref=latest`？ or we should use ` org.opencontainers.image.ref.name=latest` to specify a ref ?

And if we have multiple images for a `platform.os`, does all these images will be unpack?

Reply:
And I think in this way, we change the semantic meaning of the `ref`,  the `ref` supposed to representing a tag of image.

Reply:
>use == seems odd to me, can we just use =? 

updated.
>And another question, does it support the old way --ref=latest？ or we should use org.opencontainers.image.ref.name=latest to specify a ref ?

use org.opencontainers.image.ref.name=lates

>And if we have multiple images for a platform.os, does all these images will be unpack?

Yes, all images conforming to the platform. OS will be unpack.

Reply:
> Yes, all images conforming to the platform. OS will be unpack.

Is that a desired behavior ?  Will these images be unpack to different `rootfs` ?

Reply:
I think maybe something is not clear:
unpack at a time can only be an `image` or `imageLayout` unpack. When `ref` point to the `index`, then go to the `platform.os` to filter manifest, and then according to the manifests to unpack to the `rootfs`, I think your question is not that if the match of the manifest there are many, should not unpack to the same `rootfs`?
If this is the problem, I do not know is not able to unpack to the same file, if not, it should increase the screening of the manifests to determine.

Reply:
There could be multiple images with different ref for a specific platform in an `image` or `imageLayer`, My question is if `platform.os=xx` match multiple  image,  when you do `unpack` or `create bundle`, Will these images will be unpacked to same `rootfs`? That's not a desired behavior.

Reply:
Sorry. I did not understand it before.

>My question is if platform.os=xx match multiple image, when you do unpack or create bundle, Will these images will be unpacked to same rootfs?

No. if platform.os=xx match multiple image, The program will be [error](https://github.com/opencontainers/image-tools/pull/169/files#diff-4d7e922f1104313b276b759505857ed7R113).